Microwave.Test.Unit/BeepTest.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.VisualStudio.TestPlatform.Utilities;
+using Microwave.Classes.Boundary;
+using Microwave.Classes.Interfaces;
+using NSubstitute;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using IOutput = Microwave.Classes.Interfaces.IOutput;
+
+namespace Microwave.Test.Unit
+{
+    internal class BeepTest
+    {
+        private Beep uut;
+        private IOutput output;
+
+        [SetUp]
+        public void Setup()
+        {
+            output = Substitute.For<IOutput>();
+            uut = new Beep(output);
+        }
+
+        [Test]
+        public void CommandCalled()
+        {
+            var returned = uut.PlayBeep();
+            Assert.That(returned, Is.EqualTo(1));
+        }
+
```

Comment:
Simple class, simple test that it works. Does not test the sound part of the beep class but it would be hard to do so